charts/sysdig-deploy/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: sysdig-deploy
 description: A chart with various Sysdig components for Kubernetes
 type: application
-version: 1.55.0
+version: 1.55.1
```

Comment:
(as the subchart `node-analyzer` should be moved from 1.29.0 to 1.30.0 we should bump this chart to 1.56.0 as well)